docs/walletkit/token-provider.md

```diff
@@ -0,0 +1,264 @@
+# Docs about fetching token data and balance
+
+- Fetching token data and balance
+    - how to fetch metadata of a token
+    - how to fetch token balance of a user
+    - how to display token balance in a numerical format (i.e. bignum hex to real number)
+    - example, for now, each section need at least one
+
+Fetching token data and balance example
+
+## Introduction
+
+One of the most common usage for the data provider is to get crypto token metadata and balance of a user, the following sections state some common use cases and demonstrate how to do so.
+
+## Set up a data provider for token
```

Comment:
I am not sure this part should be in more detail(i.e. from line 17 - 33, which comes from Getting started) or just assuming the user established the factory (i.e. the version in line 34)

Reply:
This section of the documentation should have more detail, but what you have seems good for now.

Reply:
I have had a quick look, will take a look again.